server/plugin/approve.go

```diff
@@ -22,12 +22,37 @@ func (p *Plugin) httpHandleApprove(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	username := userID
```

Comment:
We can have this as empty and show it as approved by only if we were able to resolve the user id to a user name.

Approved by xchq1244 is not a good user experience.

Personally I feel we should leave it as approved if we couldn't get the user name. Not ablocker
 Approved

Reply:
You are right. Changed it :-)